src/components/Cards.js

```diff
@@ -2,17 +2,13 @@ import React from "react";
 import ApplicantLabel from "./ApplicantLabel";
 import IconContainer from "./IconContainer";
 
-const Cards = props => {
-  // The fake data instead of the data which should get from DB.
-  let callStatus = ["successful", "unsuccessful", "not yet"];
-  let applicantInfo = { applicantId: "17", applicantName: "Saeen" };
-
-  return (
-    <div>
-      <ApplicantLabel applicantInfo={applicantInfo} />
-      <IconContainer callStatus={callStatus[0]} hasMet={true} />
+const Cards = ({ students }) => {
+  return students.map(student => (
+    <div className="card">
+      <ApplicantLabel student={student} />
+      <IconContainer student={student} />
     </div>
-  );
+  ));
```

Comment:
Instead of having a `Cards` component, responsible to render multiple cards, we prefer to have a `Card` component responsible to render a single card. 
Why? Because we may want to give the _utiliser_ (the column component) the freedom to render cards ah it wants - a column, a row, a grid, with some pagination, stacked like tinder, etc.

That is: we want to delegate the `Column` component to decide how do dispose cards, meanwhile we want the `Card` to handle how its labels, icons, borders, etc are rendered

Reply:
If we want to change `Cards` to `Card`, I need to bring out the `map` method from `Cards` and do it inside `column` and assign the result array to a variable then put the variable beside `<h3>` tag inside the `return` of `Column`.

Reply:
I changed `Cards` to `Card` and made `column` more cleaner and presentational.